File: hw_2/Codes/Agcali_AlpTugrul.php
<?php
    for($test = 0; $test < 10; $test++){
        echo "test = $test \n";
        if($test == 3) break;
    }
    for($outer = 0; $outer < 10; $outer++){
        echo "outer = $outer \n";
        for($j = 0; $j < 10; $j++){
            if($outer == 2) break;
        }
    }
    echo "---------------------\n";
    for($outer = 0; $outer < 10; $outer++){
        echo "outer = $outer \n";
        for($j = 0; $j < 10; $j++){
            if($outer == 2) break 2;
        }
    }
    
?>

File: hw_2/Codes/Agcali_AlpTugrul.py
for test in range (10):
    print ("test = ", test)
    if(test == 3): break

for outer in range (10):
    print ("outer = ", outer)
    for j in range (10):
        if(outer == 2): break
    
print("----------------")
breakOut = False
for outer in range (10):
    print ("outer = ", outer)
    for j in range (10):
        if(outer == 2): 
            breakOut = True
            break
    if breakOut:
        break